packages/component-library/src/AlignedText.tsx

```diff
@@ -1,7 +1,8 @@
 import { type ComponentProps, type ReactNode, type CSSProperties } from 'react';
 
 import { Block } from './Block';
-import { View } from './View';
+
+import { View } from '@actual-app/components/view';
```

Comment:
Would you mind changing back to relative imports for these inside the "component-library" package?
Thanks

Reply:
My bad! Since most of the work was done in bulk, I didn’t notice the changes inside the component-library. I’ve reverted them